### PR TITLE
[ICDS Dashboard]: PPR Qa Fixes

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/QA-1441

https://dimagi-dev.atlassian.net/browse/QA-1434

https://dimagi-dev.atlassian.net/browse/QA-1436

https://dimagi-dev.atlassian.net/browse/QA-1427

This Bugs are resolving these things
- Removed the unlaunched states and districts from the PPD api
- Showing Not Launched in the PPR if state and district is unlaunched
- Correcting the total calculation (forgot to do an average perviously in case of quarter)
- Removing the col ```num_launched_districts``` if the report is district wise
- added missing export_info columns
- Small typo fix Aww -> AWW

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/poshan_progress_report.py [hunk 1]
```diff
@@ -149,18 +166,44 @@ def month_wise(self, filters, order_by):
...
+    def _quarter_name(self, quarter):
+        quarter_months = ''
+        if quarter == 1:
+            quarter_months = 'January-March'
+        elif quarter == 2:
+            quarter_months = 'April-June'
+        elif quarter == 3:
+            quarter_months = 'July-September'
+        elif quarter == 4:
+            quarter_months = 'July-September'
```
Comment (Rohit25negi) [inline custom/icds_reports/sqldata/exports/poshan_progress_report.py:204]: This would be oct- dec no?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left one question otherwise looks goos

File: custom/icds_reports/sqldata/exports/poshan_progress_report.py (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/sqldata/exports/poshan_progress_report.py:204]: Nice catch 👌

Comment (ShubhamBansal1997) [reply custom/icds_reports/sqldata/exports/poshan_progress_report.py]: updated!

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**